common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +99,840 @@ innovation_taming_the_skies = {
+}
+
+innovation_peacekeeper = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @peacekeeper
+
+	potential = {
+		this = culture:draenei
+	}
+
+	unlock_maa = peacekeeper
+
+	flag = global_maa
+}
+
+innovation_spellbreaker = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @spellbreaker
+
+	potential = {
+		OR = {
+			this = culture:blood_elf
+			this = culture:high_elf
+		}
+	}
+
+	unlock_maa = spellbreaker
+
+	flag = global_maa
+}
+
+innovation_subjagators = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @mogu
+
+	potential = {
+		this = culture:mogu
+	}
+
+	unlock_maa = subjagators
+
+	flag = global_maa
+}
+
+innovation_undead_ground = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @ghoul
+
+	potential = {
+		this.culture_group = culture_group:undead_group
+	}
+
+	unlock_maa = abomination
+	unlock_maa = ghoul
+
+	flag = global_maa
+}
+
+####High Medieval
+
+innovation_cloud_serpents = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @serpent
+
+	potential = {
+		this.culture_group = culture_group:pandaren_group
+	}
+
+	unlock_maa = cloud_serpents
+
+	flag = global_maa
+}
+
+innovation_scourge_aerial = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @frost_wyrm
+
+	potential = {
+		this = culture:scourge
+	}
+
+	unlock_maa = destroyer
+	unlock_maa = frost_wyrm
+
+	flag = global_maa
+}
+
+innovation_night_elf_mounted = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @huntress
+
+	potential = {
+		this = culture:night_elf
+	}
+
+	unlock_maa = huntress
+
+	flag = global_maa
+}
+
+innovation_sapper = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @sapper
+
+	potential = {
+		this = culture:goblin
+	}
+
+	unlock_maa = sapper
+
+	flag = global_maa
+}
+
+innovation_shredder = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @shredder
+
+	potential = {
+		this = culture:goblin
+	}
+
+	unlock_maa = shredder
+
+	flag = global_maa
+}
+
+innovation_steamwarrior = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @steamwarrior
+
+	potential = {
+		this = culture:gnome
+	}
+
+	unlock_maa = steam_warrior
+
+	flag = global_maa
+}
+
+innovation_technomage = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @technomage
+
+	potential = {
+		this = culture:gnome
+	}
+
+	unlock_maa = technomage
+
+	flag = global_maa
+}
+
+####Late Medival
+innovation_corsair = {
+	group = culture_group_regional
+	culture_era = culture_era_late_medieval
+	icon = @corsair
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = blunderbuss_corsairs
+
+	flag = global_maa
+}
+
+innovation_dwarven_cannon = {
```

Comment:
Don't you want to unite these?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/134936954-e1b0f21c-5a35-4ecb-a23c-361fcfa1bdc9.png)

</details>

Reply:
If you think both belong in the same era they are currently in, then yes

Reply:
> If you think both belong in the same era they are currently in, then yes

They're in the same era in-game.